.github/workflows/ci.yml

```diff
@@ -580,3 +580,31 @@ jobs:
           path: |
             ${{github.workspace}}/*-junit.xml
             ${{github.workspace}}/critestreport/*
+
+  cgroup2-misc:
```

Comment:
nit... as far as names go "CGroupsV2 and SELinux Integration" and "CGroupsV2 (misc)" are not very descriptive names :-)

Suggest:
"CGroupsV2 - SELinux enforced CRI test (macOS nested)" 
"CGroupsV2 - rootless CRI test (macOS nested)"

Reply:
Changed to `CGroupsV2 - SELinux enforced` and `CGroupsV2 - rootless CRI test`.

I don't think "macOS nested" is important here (and we don't want to use macOS just for nested VM)